tests/fixers/test_http_headers.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+from django_upgrade.data import Settings
+from tests.fixers.tools import check_noop
+from tests.fixers.tools import check_transformed
+
+settings = Settings(target_version=(4, 2))
+
+
+def test_transform_constructor_call():
+    check_transformed(
+        """\
+        from django.test import RequestFactory
+        RequestFactory("/", HTTP_HOST="test.com")
+        """,
+        """\
+        from django.test import RequestFactory
+        RequestFactory("/", headers={"host": "test.com"})
```

Comment:
This works, but it might be harder to catch cases like
```
RequestFactory().get("/", HTTP_HOST="host.com")
```
Should we handle them?

Reply:
I don't think so, we can't reliably detect common patterns of using `RequestFactory`